### PR TITLE
Fix/5

### LINKED ISSUE #5: Can't connect to port 11211
Can't connect to port 11211 if used socket

### PR DESCRIPTION
**Pull request by majormoses**
## Pull Request Checklist

fixes #5 

#### General

- [ ] Update Changelog following the conventions laid out on [Keep A Changelog](http://keepachangelog.com/)

- [ ] Update README with any necessary configuration snippets

- [ ] Binstubs are created if needed

- [ ] RuboCop passes

- [ ] Existing tests pass 

#### Purpose
Fix an issue with case sensitivity
#### Known Compatablity Issues
none

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -4,15 +4,15 @@
 
 #### General
 
-- [ ] Update Changelog following the conventions laid out on [Keep A Changelog](http://keepachangelog.com/)
+- [ ] Update Changelog following the conventions laid out on [Our CHANGELOG Guidelines ](https://github.com/sensu-plugins/community/blob/master/HOW_WE_CHANGELOG.md)
```
Comment (eheydrick) [inline .github/PULL_REQUEST_TEMPLATE.md]: I'd remove the extra space after Guidelines. Not sure if it impacts the formatting but better to keep the markdown tight.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,10 +1,17 @@
 # Change Log
 This project adheres to [Semantic Versioning](http://semver.org/).
 
-This CHANGELOG follows the format listed at [Keep A Changelog](http://keepachangelog.com/)
+This CHANGELOG follows the format listed at [Our CHANGELOG Guidelines ](https://github.com/sensu-plugins/community/blob/master/HOW_WE_CHANGELOG.md).
```
Comment (eheydrick) [inline CHANGELOG.md:4]: ditto.

**Review by eheydrick — COMMENTED** [review]
Fix looks good. Just some nitpicky formatting things.

**Comment by majormoses** [comment]
@eheydrick fixed

**Review by eheydrick — APPROVED** [review]

**Comment by swibowo** [comment]
@majormoses @eheydrick - just wondering, when are you planning to build a new version with this fix included? I'm experiencing the same issue, and I would prefer to install this fix rather than regressing to previously working v0.0.3.

**Comment by eheydrick** [comment]
Hey @swibowo, apologies for that, we aim to release quickly. I will get a release out.

**Comment by swibowo** [comment]
Thank you very much, Sir! 👍

**Comment by eheydrick** [comment]
0.1.2 is released to rubygems: https://rubygems.org/gems/sensu-plugins-memcached/versions/0.1.2. Thanks again for your patience.

**Pull request merged**